lib/book.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'stdio'
+
+class Book
+  attr_accessor :title, :author
+
+  @instances = []
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+    self.class.all << self
+  end
+
+  def rent_book(rental)
+    @rentals.push(rental)
+  end
+
+  def self.all
+    @instances
+  end
+
+  def self.create
+    title, author = StandardIO.book_prompt
```

Comment:
- [ ] You should follow the dependency inversion principle. Your classes should not depend on another module or class.